### PR TITLE
Some improvements

### LINKED ISSUE #53: add logo

### PR DESCRIPTION
**Pull request by c0m1t**
It closes following issues:
- Add Sahab logo to nav-bar #53 
- Add description for pages #60 
- Add text-area label in strengths-weaknesses page #61 
- Add text-are label in criteria page #65
- Change maxChars limit of distinct characteristics to 128 #88  

And also has some minor improvements:
- Move add buttons in strengths-weaknesses page to the left and changed the variant to outlined
- Change variant of add project button in Projects page
- Change background-color of the app to grey
- Add margin top to sticky-action-bar component
- Update links text in nav-bar
- Use i18n and mdx

### CODE REVIEW

File: src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx [hunk 1]
```diff
@@ -12,37 +12,60 @@ import LimitedTextAreaInput from '../forminator/inputs/LimitedTextAreaInput';
...
+export function StrengthsOrWeaknesses({ title, maxLength, label, ...props }: Props) {
   const lens = useFragmentLens();
-  const condition = useCallback(
+  const firstCondition = useCallback((value: unknown[]) => {
+    return value ? value.length < 2 : true;
+  }, []);
+  const secondCondition = useCallback(
     (value: unknown[]) => {
-      return value ? value.length < maxLength : true;
+      return value ? value.length > 1 && value.length < maxLength : false;
```
Comment (m-davoodi) [inline src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx]:
```js
return value && value.length > 1 && value.length < maxLength;
```

File: src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx [hunk 2]
```diff
@@ -12,37 +12,60 @@ import LimitedTextAreaInput from '../forminator/inputs/LimitedTextAreaInput';
...
+                  <LimitedTextAreaInput
+                    variant="outlined"
+                    maxChars={280}
+                    fullWidth
+                    inputProps={{ dir: 'auto' }}
+                    label={label}
+                  />
+                </Grid>
+              </ArrayOutput>
+              <ConditionalSection condition={firstCondition} lens={lens}>
```
Comment (m-davoodi) [inline src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx]: Is it possible to refactor codes  something like this:
```jsx
<Grid item xs />
<Grid item>
  <ArrayAppendButton variant="outlined" color="primary">
    {firstCondition() && i18n._('Add')  || secondCondition() && i18n._('What else')}
  </ArrayAppendButton>
</Grid>
```

Comment (c0m1t) [reply src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx]: i dont think so cuz the button is not always displayed. we only want to display it when value.length is less than maxLength

File: src/containers/main/MainContainer.tsx [hunk 3]
```diff
@@ -16,26 +16,27 @@ import { FullPageSpinner } from 'src/shared/loading';
...
+import sahabLogo from 'src/assets/sahab-logo.png';
 
 interface OwnProps {}
 
 type Props = FCProps<OwnProps>;
 
 const items: MenuItem[] = [
   {
-    text: i18n._('Criteria'),
+    text: 'شایستگی‌های عملکردی',
```
Comment (smmoosavi) [inline src/containers/main/MainContainer.tsx]: i18n

File: src/containers/main/MainContainer.tsx [hunk 4]
```diff
@@ -16,26 +16,27 @@ import { FullPageSpinner } from 'src/shared/loading';
...
     link: {
       to: '/criteria',
     },
   },
   {
-    text: i18n._('Strengths and Weaknesses'),
+    text: 'ویژگی‌های بارز',
```
Comment (smmoosavi) [inline src/containers/main/MainContainer.tsx]: i18n

File: src/containers/main/MainContainer.tsx [hunk 5]
```diff
@@ -16,26 +16,27 @@ import { FullPageSpinner } from 'src/shared/loading';
...
     link: {
       to: '/strengths-weaknesses',
     },
   },
   {
-    text: i18n._('Projects'),
+    text: 'دستاوردها',
```
Comment (smmoosavi) [inline src/containers/main/MainContainer.tsx]: i18n

File: src/pages/criteria-page/CriteriaDescriptionCard.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+import { FCProps } from 'src/shared/types/FCProps';
+
+interface OwnProps {}
+
+type Props = FCProps<OwnProps>;
+
+export function CriteriaDescriptionCard(props: Props) {
+  return (
+    <Card>
+      <CardHeader title="مرور شایستگی‌های عملکردی" />
```
Comment (smmoosavi) [inline src/pages/criteria-page/CriteriaDescriptionCard.tsx]: i18n

File: src/pages/criteria-page/CriteriaPage.tsx [hunk 7]
```diff
@@ -80,7 +80,7 @@ export default function CriteriaPage(props: Props) {
 
   return (
     <Suspense fallback={FullPageSpinner}>
-      <Container maxWidth="sm">
+      <Container maxWidth="md">
         <Box marginY={2}>
           <CriteriaForm
```
Comment (smmoosavi) [inline src/pages/criteria-page/CriteriaPage.tsx]: نباید در این نقطه 

Container

دیگری هم باشد؟

File: src/pages/projects-page/ProjectsDescriptionCard.tsx [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+type Props = FCProps<OwnProps>;
+
+export function ProjectsDescriptionCard(props: Props) {
+  return (
+    <Card>
+      <CardHeader title="مرور دستاوردها" />
+      <CardContent>
+        <Container maxWidth="sm">
+          <Typography>
+            توی این بخش تمرکزمون میخواد روی دستاورد ها و اثری که طی یک سال گذشته روی پروژه‌ها داشتیم باشه. برای همین از
```
Comment (smmoosavi) [inline src/pages/projects-page/ProjectsDescriptionCard.tsx]: use mdx

File: src/pages/projects-page/ProjectsDescriptionCard.tsx [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+import { FCProps } from 'src/shared/types/FCProps';
+
+interface OwnProps {}
+
+type Props = FCProps<OwnProps>;
+
+export function ProjectsDescriptionCard(props: Props) {
+  return (
+    <Card>
+      <CardHeader title="مرور دستاوردها" />
```
Comment (smmoosavi) [inline src/pages/projects-page/ProjectsDescriptionCard.tsx]: use i18n

File: src/pages/strengths-weaknesses-page/StrengthsWeaknessDescriptionCard.tsx [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+import { FCProps } from 'src/shared/types/FCProps';
+
+interface OwnProps {}
+
+type Props = FCProps<OwnProps>;
+
+export function StrengthsWeaknessDescriptionCard(props: Props) {
+  return (
+    <Card>
+      <CardHeader title="مرور ویژگی‌های بارز" />
```
Comment (smmoosavi) [inline src/pages/strengths-weaknesses-page/StrengthsWeaknessDescriptionCard.tsx]: i18n

File: src/pages/strengths-weaknesses-page/StrengthsWeaknessDescriptionCard.tsx [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+      <CardContent>
+        <Container maxWidth="sm">
+          <Typography>
```
Comment (smmoosavi) [inline src/pages/strengths-weaknesses-page/StrengthsWeaknessDescriptionCard.tsx]: use mdx

File: src/pages/strengths-weaknesses-page/StrengthsWeaknessesForm.tsx [hunk 12]
```diff
@@ -21,15 +22,26 @@ export function StrengthsWeaknessesForm(props: Props) {
...
         <DictInput>
+          <Grid item xs={12}>
+            <StrengthsWeaknessDescriptionCard />
+          </Grid>
           <DictInputItem field="strengths">
             <Grid item xs={12}>
-              <StrengthsOrWeaknesses maxLength={3} title={i18n._('Strengths')} />
+              <StrengthsOrWeaknesses
+                maxLength={3}
+                title="مهمترین ویژگی ها یا رفتارهای مؤثری که باید ادامه شون بدم"
```
Comment (smmoosavi) [inline src/pages/strengths-weaknesses-page/StrengthsWeaknessesForm.tsx]: i18n

File: src/pages/strengths-weaknesses-page/StrengthsWeaknessesForm.tsx [hunk 13]
```diff
@@ -21,15 +22,26 @@ export function StrengthsWeaknessesForm(props: Props) {
...
+              />
             </Grid>
           </DictInputItem>
 
           <DictInputItem field="weaknesses">
             <Grid item xs={12}>
-              <StrengthsOrWeaknesses maxLength={3} title={i18n._('Weaknesses')} />
+              <StrengthsOrWeaknesses
+                maxLength={3}
+                title="مهمترین ویژگی ها یا رفتارهایی که باید باید توی خودم بهبودشون بدم"
```
Comment (smmoosavi) [inline src/pages/strengths-weaknesses-page/StrengthsWeaknessesForm.tsx]: i18n

File: src/locales/fa/messages.po [hunk 14]
```diff
@@ -38,7 +43,10 @@ msgstr "آیا از حذف  این پروژه مطمین هستید؟"
 msgid "Cancel"
 msgstr "لغو"
 
-#: src/containers/main/MainContainer.tsx:11
+#: src/pages/criteria-page/CriteriaForm.tsx:32
+msgid "Collaboration"
```
Comment (smmoosavi) [inline src/locales/fa/messages.po]: NAGT: Not A Good Translation

File: src/locales/fa/messages.po [hunk 15]
```diff
@@ -80,6 +101,14 @@ msgstr "ورود"
 msgid "Meets Expectations"
 msgstr "مطابق با انتظارات"
 
+#: src/pages/strengths-weaknesses-page/StrengthsWeaknessesForm.tsx:24
+msgid "Most important characteristics or behaviours I should improve in myself"
```
Comment (smmoosavi) [inline src/locales/fa/messages.po]: behaviors

File: src/pages/criteria-page/ExecutionContent.mdx [hunk 16]
```diff
@@ -0,0 +1 @@
+انجام کار با کیفیت با حداقل راهنمایی. برنامه‌ریزی و زمان بندی. حفظ توازن سرعت و دقت. قدرت تحلیل الگوریتمی. مستند سازی و ارتباط نوشتاری واضح، منسجم و دقیق. حجم کار Done شده.
```
Comment (smmoosavi) [inline src/pages/criteria-page/ExecutionContent.mdx]: زمان‌بندی

Comment (smmoosavi) [reply src/pages/criteria-page/ExecutionContent.mdx]: مستندسازی

File: src/shared/criterion-item/CriterionItem.tsx [hunk 17]
```diff
@@ -8,33 +8,35 @@ import { FCProps } from 'src/shared/types/FCProps';
 interface OwnProps {
   title: string;
   prefix: string;
-  details?: string;
+  details?: React.FunctionComponent;
```
Comment (smmoosavi) [inline src/shared/criterion-item/CriterionItem.tsx]: reactNode

File: src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx [hunk 18]
```diff
@@ -12,37 +12,49 @@ import LimitedTextAreaInput from '../forminator/inputs/LimitedTextAreaInput';
...
 }
 
 type Props = FCProps<OwnProps>;
 
-export function StrengthsOrWeaknesses({ title, maxLength, ...props }: Props) {
+export function StrengthsOrWeaknesses({ title, maxLength, label, ...props }: Props) {
   const lens = useFragmentLens();
   const condition = useCallback(
     (value: unknown[]) => {
-      return value ? value.length < maxLength : true;
```
Comment (sijav) [inline src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx]: It's nice to revert this as `value ? condition : true` is much more readable than `!value || condition`

Comment (sijav) [inline src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx]: It's nice to revert this as value ? condition : true is much more readable than !value || condition;

**Pull request merged**